web/wp-content/themes/dctx/header.php

```diff
@@ -11,6 +11,16 @@
 	<link rel="icon" type="image/png" href="<?php echo ROOT; ?>favicon.png">
```

Comment:
:no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'ROOT' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

Reply:
skip as done from prev